frontend/server/src/Controllers/Run.php

```diff
@@ -1051,16 +1208,19 @@ public static function apiDownload(\OmegaUp\Request $r): void {
         }
         // Get the user who is calling this API
         $r->ensureIdentity();
+        $showDiff = $r->ensureOptionalBool('show_diff') ?? false;
 
         \OmegaUp\Validators::validateStringNonEmpty(
             $r['run_alias'],
             'run_alias'
         );
+
         if (
             !self::downloadSubmission(
                 $r['run_alias'],
                 $r->identity,
-                /*passthru=*/true
+                /*$passthru=*/true,
+                $showDiff
```

Comment:
```suggestion
                /*$skipAuthorization=*/$showDiff
```

aquí es mejor agregar una pista para el lector del efecto que va a tener esta bandera.

Reply:
Listo